api/Models/FilterParameters.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace iTechArt.Hotels.Api.Models
+{
+    public class FilterParameters
+    {
+        [MaxLength(60)]
+        public string Name { get; set; }
+
+        [MaxLength(56)]
+        public string Country { get; set; }
```

Comment:
Why exactly 56? :)

Reply:
I knew you would ask))
![image](https://user-images.githubusercontent.com/54571357/148740364-8195a9dc-f623-45a2-abf5-8aa892835fda.png)

The same approach for country field inside hotel

Reply:
What if we have new country or some of existing changes its name?

Reply:
Well, I could not foresee future country names. If to talk about the longest one now, it could be shortened like UK. And I also think that any county name might be shortened to fit into 56 symbols